tutorials/algorithms/06_grover.ipynb

```diff
@@ -371,19 +370,19 @@
       "text/plain": [
-       "<Figure size 146.797x325.08 with 1 Axes>"
+       "<Figure size 203.885x451.5 with 1 Axes>"
       ]
      },
-     "execution_count": 8,
+     "execution_count": 48,
      "metadata": {},
      "output_type": "execute_result"
     }
```

Comment:
Similarly, here `oracle.ccz(0, 1, 2)` could be a bit more straightforward to use than `oracle.append(ZGate().control(2), [0, 1, 2])`